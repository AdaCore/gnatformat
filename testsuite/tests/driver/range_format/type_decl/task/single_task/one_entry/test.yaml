description: SingleTaskDecl formatting for line selection.
driver: gnatformat
args:
  [
    "test.ads",
    "--range-format",
    "-SL 129",
    "-EL 130",
    "--pipe",
  ]
control:
- [XFAIL, "True",
   "unexpected output: waiting for issue #eng/libadalang/libadalang#1413"]