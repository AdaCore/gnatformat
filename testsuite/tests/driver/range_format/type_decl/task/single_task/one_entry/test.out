test.ads(<SingleTaskDecl ["Controller"] test.ads:129:4-131:19>) - 129:1-131:19
^
   task Controller is
      entry Request (Level)(D : Item);  --  a family of entries
   end Controller;$

